### ISSUE TITLE
Feature request: detect osm objects with a wikipedia tag or wikidata tag but not both

### ISSUE THREAD

**Issue by brierjon** [osm-fr/osmose-backend#191]
Some OSM objects are missing a complementary tag. Some objects have wikipedia articles, but are missing a wikidata tag.

The wikidata tag and value should be present whenever a wikipedia tag and value is present and a wikipedia tag and value should be present when a wikidata tag and value are present.

There are OSM objects that currently suffer from the missing complementary fields.

**Comment by frodrigo** [comment]
Can you provide link to extract of wikipedia/wikidata data so Osmose can suggest quick fix ?

**Comment by brierjon** [comment]
I'm not sure what extract format is being requested. Would you clarify if the following is insufficient.

First example: Q148554 maps to en:National Museum of Natural History in a Wikidata to Wikipedia relationship In some cases the Wikidata Q148554 may be missing when a Wikipedia title is available en:National Museum of Natural History.

Second example of a good value: Berlin - https://www.wikidata.org/wiki/Q64 and https://www.openstreetmap.org/relation/62422 which should link to https://de.wikipedia.org/wiki/Berlin (There are 242 Wikipedia pages in other languages for this object and a reason why Wikidata is important to link to objects in OSM as any of the 243 pages could be returned for the OSM object / relationship.)

Look for a Wikipedia entry without https://www.wikidata.org/wiki/Property:P402. While an existing P402 should allow linking to Wikipedia entries linked to the item P402 is associated. As noted in [https://wiki.openstreetmap.org/wiki/Wikidata](https://wiki.openstreetmap.org/wiki/Wikidata)

Preventing future issues is being worked on for the OSM iD - https://github.com/openstreetmap/iD/issues/3929#issuecomment-310736467

**Comment by frodrigo** [comment]
My question was about a data base dump from wikipedia/wikidata to be able to suggest the change.

**Comment by brierjon** [comment]
You can get a dump for Wikidata can be grabbed from: [https://www.wikidata.org/wiki/Wikidata:Database_download](https://www.wikidata.org/wiki/Wikidata:Database_download) 

There are dynamic modes too: 
[https://www.wikidata.org/wiki/Wikidata:Data_access](https://www.wikidata.org/wiki/Wikidata:Data_access)

**Comment by matkoniecz** [comment]
I strongly disagree with this feature request. This kind of issue is easily fixable with a mechanical edit and asking human editors to do that is a huge waste of time.

If some community wants this type of edit to be run please contact me (Mateusz Konieczny username at OSM) - I made such program and run it in Poland - see https://wiki.openstreetmap.org/wiki/Mechanical_Edits/Mateusz_Konieczny_-_bot_account for details.
